### PR TITLE
Issue 3517

### LINKED ISSUE #3517: LocalServiceManagerTest: Unused import 'import org.assertj.core.api.InstanceOfAssertFactory;'
### Description

The class [LocalServiceManagerTest](https://github.com/eclipse-jkube/jkube/blob/e398760a1d0067661c3f33b7b7d19150442a47c3/jkube-kit/remote-dev/src/test/java/org/eclipse/jkube/kit/remotedev/LocalServiceManagerTest.java#L26) presents the issue: Unused import 'import org.assertj.core.api.InstanceOfAssertFactory;'.

The following line:
https://github.com/eclipse-jkube/jkube/blob/e398760a1d0067661c3f33b7b7d19150442a47c3/jkube-kit/remote-dev/src/test/java/org/eclipse/jkube/kit/remotedev/LocalServiceManagerTest.java#L26
should be removed.
### :red_circle: Before you start :red_circle:

:point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down:

Make sure you read the [contributing guide](https://www.eclipse.org/jkube/contributing) first.

Pay special attention to the ECA agreement section and the requirement to sign-off your commit.

This is a [first-timers-only](https://eclipse.dev/jkube/contributing/#first-timers-only-issues) issue. These issues are especially created for contributors who are contributing to the project for the **first time**.

If you have already contributed to this project in the past, please don't ask to be assigned one of these issues and look for [good-first-issues](https://github.com/eclipse-jkube/jkube/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) or [help-wanted](https://github.com/eclipse-jkube/jkube/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22+) issues instead in order to continue you contributor journey in the project.

### PR DESCRIPTION
**Pull request by abhishektripathi66**
## Description

Fixes #3517 
This PR resolves an issue of have an unused Imports in the Test class. 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.dev/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] My code follows the style guidelines of this project
 - [x] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [x] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [x] I have performed a self-review of my code
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [x] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [x] I have added tests that prove my fix is effective or that my feature works
 - [x] New and existing unit tests pass locally with my changes
 - [x] I tested my code in Kubernetes
 - [x] I tested my code in OpenShift

### CODE REVIEW

**Comment by manusa** [comment]
### Eclipse JKube [CI Report](https://github.com/eclipse-jkube/ci)

Started new GH workflow run for https://github.com/eclipse-jkube/jkube/pull/3533 (_2024-11-13T11:07:42Z_)

:gear: [JKube E2E Tests (11816003986)](https://github.com/eclipse-jkube/ci/actions/runs/11816003986) :hourglass_flowing_sand:
- :hourglass_flowing_sand: [Bootstrap CI test run (#3533)](https://github.com/eclipse-jkube/ci/actions/runs/11816003986/job/32918385920)

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Usage:
 * Fix #2667: Add new helm test goal task (`k8s:helm-test` for maven and `k8sHelmTest` for gradle)
 * Fix #3326: Micronaut healthcheck enricher infers overridden server port in application.properties
 * Fix #3354: Build fails with `imageStream` for `buildRecreate` value
+* Fix #3517: LocalServiceManagerTest: Unused import 'import org.assertj.core.api.InstanceOfAssertFactory;
```
[the remark this answers is not in the document]
Comment (abhishektripathi66) [reply CHANGELOG.md:30]: Sure, will remove it

[the remark this answers is not in the document]
Comment (abhishektripathi66) [reply CHANGELOG.md]: Done @rohanKanojia

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**